lib/deqm_test_kit/care_gaps.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module DEQMTestKit
+  # tests for $care-gaps
+  class CareGaps < Inferno::TestGroup
+    id 'care_gaps'
+    title 'Gaps in Care'
+    description 'Ensure FHIR server can calculate gaps in care for a measure'
+
+    fhir_client do
+      url :url
+    end
+
+    INVALID_ID = 'INVALID_ID'
+
+    test do
+      title 'Check $care-gaps proper calculation'
+      id 'care-gaps-01'
+      description 'Server should properly return a gaps report'
+      input :measure_id, :patient_id
```

Comment:
This is probably out of scope for your task, but for future tasking, should we have something like a patient availability test for the patient id (similar to measure availability)?